content/en/Integrations/IntegrationBuilder/How to Guides/add-pentest-title-to-jira-issue-summary.md

```diff
@@ -0,0 +1,80 @@
+---
+title: "Add Pentest Title to Jira Issue Summary"
+linkTitle: "Add Pentest Title to Jira Issue Summary"
+weight: 70
+---
+
+{{% pageinfo %}}
+
+This guide is designed to help [**Integration Builder**](/integrations/integrationbuilder/) users customize the summary field of 
+Jira issues created by their recipes to contain the title of the relevant pentest.
+Please follow the [Create Jira tickets for Findings guide](/integrations/integrationbuilder/how-to-guides/jira-cloud-migration) before proceeding with this guide.
+
+{{% /pageinfo %}}
+
+1. Navigate to the __[Cobalt > Jira Cloud] Push pentest finding to Jira__ recipe that you set up while [following the previous guide](/integrations/integrationbuilder/how-to-guides/jira-cloud-migration).
+
+2. Open the edit view for the recipe by clicking the __Edit__ button.
+
+3. Hover your cursor above the __Monitor__ block that encapsulates the __Create issue in Jira__ action. An __Add step__ button will appear. Click on it.
+
+![add step](/integrations/integration_builder/how_to_guides/add-pentest-title-to-jira-issue-summary/1_add_step.png)
```

Comment:
I recommend changing the markdown for embedded images to use our new partial, which is used in the other pages. For example
```suggestion
{{% image src="/integrations/integration_builder/how_to_guides/add-pentest-title-to-jira-issue-summary/1_add_step.png" alt="add step" %}}
```
This will create a better frame and shadow around the images to stand out from the surrounding text.

Reply:
Very nice. Updated. Thanks!